tests/tests/Data/RawCodeCoverageDataTest.php

```diff
@@ -361,7 +361,6 @@ public function testHeavyIndentationIsHandledCorrectly(): void
                 99,
                 101,
                 116,
-                117,
```

Comment:
:confetti_ball: Hurray that this also fixes a harmless but tedious long lasting bug :nail_care: